core/logger/logger.go

```diff
@@ -241,8 +269,14 @@ func (c *Config) New() (Logger, func() error) {
 	if err != nil {
 		log.Fatal(err)
 	}
+
+	// If Splunk logging is enabled (token set), extend/wrap the logger with a splunkLogger instance
+	if c.SplunkToken != "" {
+		l = newSplunkLogger(l, c.SplunkToken, c.SplunkURL, c.Hostname, c.ChainlinkDev)
```

Comment:
Perhaps we should wrap this one last, so that when it makes internal log calls, they will be going through the other wrappers - otherwise failures to post to splunk would not be counted/reported through prometheus and sentry like other calls.

Reply:
That's a good point, swapped the order